gamemode/modules/base/sv_purchasing.lua

```diff
@@ -123,7 +123,7 @@ function DarkRP.hooks:canBuyShipment(ply, shipment)
     end
 
     local canbecome = false
-    for _, b in pairs(shipment.allowed) do
+    for _, b in ipairs(shipment.allowed) do
```

Comment:
This is a shipment, while not a custom team, it _is_ modified by addon developers and server owners